lib/mayaUsd/utils/hash.h

```diff
@@ -28,7 +29,7 @@ namespace MAYAUSD_NS_DEF {
 // boost::hash implementation also relies on the same algorithm:
 // https://www.boost.org/doc/libs/1_64_0/boost/functional/hash/hash.hpp
 
-template <typename T> inline void hash_combine(size_t& seed, const T& value)
+template <typename T> inline void hash_combine(std::size_t& seed, const T& value)
```

Comment:
.../lib/mayaUsd/utils/hash.h:31:48: error: ‘size_t’ was not declared in this scope; did you mean ‘std::size_t’?